### ISSUE TITLE
TFLite benchmark tool with default cmake build shows weird time duration on DEPTHWISE_CONV_2D 

### ISSUE THREAD

**Issue by captainst** [tensorflow/tensorflow#61950]
### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

tf 2.9.3, tf 2.13.0

### Custom code

No

### OS platform and distribution

Linux Ubuntu 20.04

### Bazel version

5.0

### GCC/compiler version

gcc 9.4.0

### Current behavior?

Following the guide for [build_cmake](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/guide/build_cmake.md)
`cmake ../tensorflow_src/tensorflow/lite`
then build the benchmark tool & label_image:
```
cmake --build . -j -t benchmark_model
cmake --build . -j -t label_image
```
I tested using the model [here](https://storage.googleapis.com/download.tensorflow.org/models/tflite/mobilenet_v1_1.0_224_quant_and_labels.zip).

The results shows:
============================== Summary by node type ==============================
	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
	                 CONV_2D	       15	    38.265	    58.415%	    58.415%	     0.000	       15
	       DEPTHWISE_CONV_2D	       13	    27.230	    41.569%	    99.985%	     0.000	       13
	         AVERAGE_POOL_2D	        1	     0.007	     0.011%	    99.995%	     0.000	        1
	                 SOFTMAX	        1	     0.003	     0.005%	   100.000%	     0.000	        1
	                 RESHAPE	        1	     0.000	     0.000%	   100.000%	     0.000	        1

In theory, 13 x DEPTHWISE_CONV_2D should consume MUCH less time than 15 CONV_2, but it is not the case here.

However, if I use the pre-built binary from [here](https://storage.googleapis.com/tensorflow-nightly-public/prod/tensorflow/release/lite/tools/nightly/latest/linux_x86-64_benchmark_model), the result is correct.

### Standalone code to reproduce the issue

```shell
benchmark_model --graph=./mobilenet_quant_v1_224.tflite --enable_op_profiling=true
```

### Relevant log output

```shell
============================== Summary by node type ==============================
	             [Node type]	  [count]	  [avg ms]	    [avg %]	    [cdf %]	  [mem KB]	[times called]
	                 CONV_2D	       15	    38.265	    58.415%	    58.415%	     0.000	       15
	       DEPTHWISE_CONV_2D	       13	    27.230	    41.569%	    99.985%	     0.000	       13
	         AVERAGE_POOL_2D	        1	     0.007	     0.011%	    99.995%	     0.000	        1
	                 SOFTMAX	        1	     0.003	     0.005%	   100.000%	     0.000	        1
	                 RESHAPE	        1	     0.000	     0.000%	   100.000%	     0.000	        1
```

**Comment by captainst** [comment]
#61946

**Comment by pjpratik** [comment]
Hi @captainst 

Sorry for the delayed response.

Have you observed the same in multiple runs, also can you check the same if it replicates with nightly pull as well?

Thanks.
